src/component/counter.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+class Counter extends Component {
+    state = {  
+        count: 0
+    };
+
+    styles = {
+        fontSize: 30,
+        fonWeight: 'bold'
+    };
+    render() { 
+        return ( 
+            <div>
+                <span style={this.styles} class="badge badge-pill badge-success m-2">{this.state.count}</span>
```

Comment:
cái this.state.count này em có thể destructor ra cái biến count được không

Reply:
dạ em có làm rồi ạ. Phần này em mới tạo app test thử thôi ạ.

Reply:
Đoạn sau em có chỉnh lại thành destructuring argument rồi mà chưa commit lên ạ.